core/src/Telemetry/HttpEvent.cs

```diff
@@ -77,5 +83,34 @@ public string OauthErrorCode
         {
             set { this[OauthErrorCodeKey] = value; }
         }
+
+        public string HttpMethod
+        {
+            set { this[HttpMethodKey] = value;  }
+        }
+
+        public string RequestIdHeader
+        {
+            set { this[RequestIdHeaderKey] = value; }
+        }
+        public string TokenAge
+        {
+            set { this[TokenAgeKey] = value; }
+        }
+
+        public string SpeInfo
```

Comment:
>SpeInfo [](start = 22, length = 7)

What's SpeInfo?

Reply:
I'm curious about this one as well :)

---
In reply to: [228347941](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1352#discussion_r228347941) [](ancestors = 228347941)

Reply:
SpeInfo is the server's version of deployment group/ring

Reply:
Please comment/document the meaning of this field.   Links to telemetry/server documentation?